File: VisaD.Application/Applications/Dtos/ApplicationLotHistoryDto.cs
using System.Collections.Generic;

namespace VisaD.Application.Applications.Dtos
{
	public class ApplicationLotHistoryDto
	{
		public List<ApplicationCommitHistoryItemDto> Commits { get; set; } = new List<ApplicationCommitHistoryItemDto>();

		public int ActualCommitId { get; set; }
	}
}

File: WebApplication/src/modules/application/models/application-lot-history.dto.ts
import { ApplicationCommitHistoryItemDto } from './application-commit-history-item.dto';
import { ApplicationLotResultDto } from './application-lot-result.dto';

export class ApplicationLotHistoryDto {
  hasResult: boolean;
  result: ApplicationLotResultDto;
  commits: ApplicationCommitHistoryItemDto[];
  actualCommitId: number;
}
